.gitignore

```diff
@@ -21,3 +21,4 @@ bin/fabfile.py
 
 # other
 *.pyc
+var/
```

Comment:
`app/cache` and `app/logs` don't need to be ignored anymore

Reply:
Only sessions are in var atm